search/includes/classes/class-settingshealthjob.php

```diff
@@ -447,50 +461,35 @@ protected function check_process_build() {
 	}
 
 	/**
-	 * Activate new index version and delete old one.
+	 * Alert to activate new index version and delete old one.
 	 *
 	 * @param object $Indexable Indexable object we want to swap out
 	 */
-	public function swap_index_versions( $indexable ) {
-		$activate_version = $this->search->versioning->activate_version( $indexable, 'next' );
-		if ( is_wp_error( $activate_version ) ) {
-			$message = sprintf(
-				'Application %s: An error occurred during activation of new %s index on %s for shard requirements: %s',
-				FILES_CLIENT_SITE_ID,
-				$indexable->slug,
-				home_url(),
-				$activate_version->get_error_message()
-			);
-			$this->send_alert( '#vip-go-es-alerts', $message, 2 );
-
-			return;
-		}
-
-		$delete_version = $this->search->versioning->delete_version( $indexable, 'previous' );
-		if ( is_wp_error( $delete_version ) ) {
+	public function alert_to_swap_index_versions( $indexable ) {
+		$old_version = $this->search->versioning->get_active_version_number( $indexable );
+		$new_version = $this->search->versioning->get_inactive_version_number( $indexable );
+		if ( is_wp_error( $new_version ) ) {
 			$message = sprintf(
-				'Application %s: An error occurred during deletion of old %s index on %s for shard requirements: %s',
+				'Application %s: An error occurred locating the newly built index on %s for shard requirements (%s)',
 				FILES_CLIENT_SITE_ID,
 				$indexable->slug,
 				home_url(),
-				$delete_version->get_error_message()
+				$new_version->get_error_message()
 			);
 			$this->send_alert( '#vip-go-es-alerts', $message, 2 );
 
 			return;
 		}
 
-		\Automattic\VIP\Logstash\log2logstash(
```

Comment:
What's the intent here with this removal? I think it's still good to have that info for audit.

Reply:
It's not being removed, it was just being moved up for right after the indexing completes